Build/Common.props

```diff
@@ -17,7 +17,7 @@
       as it will restart file versions so 2.4.0-beta1 may have higher 
       file version (like 2.4.0.2222) than 2.4.0-beta2 (like 2.4.0.1111)
     -->
-    <SemanticVersionDate>2019-03-16</SemanticVersionDate>
+    <SemanticVersionDate>2021-06-20</SemanticVersionDate>
```

Comment:
maybe update this to latest date since this pr has been open a while?

Reply:
This is for local only so the date holds little significance.